packages/design-system/src/components/chip/chip.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import { useEffect, useState } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import Chip from './chip';
+
+interface ChipProps {
+  label: string;
+  variant?: 'home' | 'search';
+  isActive?: boolean;
+  onDelete?: () => void;
+  onScrollToSection?: () => void;
+  onActiveChange?: (label: string) => void;
+}
+
+const meta: Meta<typeof Chip> = {
+  title: 'Common/Chip',
+  component: Chip,
+  parameters: {
+    layout: 'centered',
+  },
+  tags: ['autodocs'],
+  args: {
+    label: '기본 칩',
+  },
+  argTypes: {
+    onDelete: { table: { disable: true } },
+    onScrollToSection: { table: { disable: true } },
+    onActiveChange: { table: { disable: true } },
+    isActive: { control: 'boolean' },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof Chip>;
+
+const HomeTemplate = (args: ChipProps) => {
+  const [activeChip, setActiveChip] = useState<string | null>(null);
```

Comment:
```suggestion
 const [activeChip, setActiveChip] = useState(null);
```

이렇게 작성해도 string | null 로 타입추론이 가능할 것 같아요